app/src/main/java/com/group80/uoftinder/CreateAccountView.java

```diff
@@ -268,9 +271,216 @@ public void onClick(View v) {
         });
     }
 
+    /**
+     * Creates the friendship_questionnaire.xml and gets inputs, proceeds to recommendation page if
+     * information is entered correctly
+     * @param currentUser   the current user trying to register
+     */
+    private void createFriendshipQuestionnaire(User currentUser) {
```

Comment:
Have you considered making one ```createQuestionnnaire``` method that takes in a User? You could figure out the type from the ```currentUser``` object.